prime-dotnet-webapi/Configuration/GlobalClauseConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Prime.Models;
+
+public class GlobalClauseConfiguration : IEntityTypeConfiguration<GlobalClause>
+{
+    private readonly Guid SYSTEM_USER = Guid.Empty;
+
+    private readonly DateTime SEEDING_DATE = DateTime.Now;
+
+    public void Configure(EntityTypeBuilder<GlobalClause> builder)
+    {
+        #region GlobalClauseSeed
```

Comment:
#region is a directive for visual studio to collapse the section for easier reading, it has not effect on the code. I assume you added these lines to be consistent with the other code, but we are removing them in another PR.

Reply:
K I'll remove them

Reply:
Removed